.gitignore

```diff
@@ -273,3 +273,4 @@ $RECYCLE.BIN/
 
 # End of https://www.toptal.com/developers/gitignore/api/python,macos,linux,windows,visualstudiocode
 .vscode/settings.json
+.envrc
```

Comment:
Lots of controversy on this one (because it is so trivial of an issue, like where to put `{` in your code), but you should add `.envrc` to your global gitignore rather than on every project.